stdlib/src/utils/static_tuple.mojo

```diff
@@ -404,3 +405,15 @@ struct InlineArray[ElementType: CollectionElement, size: Int](Sized):
             normalized_idx += size
 
         return self[]._get_reference_unsafe(normalized_idx)
+
+    @always_inline
+    fn unsafe_ptr(
+        self: Reference[Self, _, _]
+    ) -> UnsafePointer[Self.ElementType]:
+        """Returns a reference to the data. Use with caution.
+
+        Returns:
+            A pointer to the data.
+        """
+        # YOLO, we're crazy people. This is highly unsafe.
+        return UnsafePointer(self).bitcast[Self.ElementType]()
```

Comment:
```suggestion
        return UnsafePointer(self._array).bitcast[Self.ElementType]()
```

**Suggestion** Make it more explicit that this is a pointer to the underlying `!pop.array` value. 

This is actually pretty safe (as far as unsafe operations go) 🙂 

What would NOT be very safe is using `pop.array.gep` to the 0th element and then creating offset pointers from that—I know from experience with `_ArrayMem` that that misleads the the Mojo compiler and leads to strange UB-like results.

Reply:
I ran into this footgun too, getting the 0th element was NOT the right called. I lost one hour with this XD